akka-docs/rst/java/code/docs/http/javadsl/server/directives/RouteDirectivesExamplesTest.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (C) 2016-2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package docs.http.javadsl.server.directives;
+
+import akka.http.javadsl.model.HttpEntities;
+import akka.http.javadsl.model.HttpRequest;
+import akka.http.javadsl.model.Uri;
+import akka.http.javadsl.model.headers.ContentType;
+import akka.http.javadsl.model.ContentTypes;
+import akka.http.javadsl.model.HttpResponse;
+import akka.http.javadsl.model.StatusCodes;
+import akka.http.javadsl.server.Rejections;
+import akka.http.javadsl.server.Route;
+import akka.http.javadsl.testkit.JUnitRouteTest;
+import org.junit.Test;
+
+import java.util.Collections;
+
+/**
+ * @author Hawstein
+ */
```

Comment:
Sorry for that, trust me I definitely know the rule that not to use the `@author` tags and remove it in the previous PRs. I just forgot to remove it in this PR and will fix it now.